spring-batch-plus-sample/spring-batch-plus-kotlin-dsl/build.gradle.kts

```diff
@@ -17,12 +17,12 @@ repositories {
 tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
     kotlinOptions {
         freeCompilerArgs = listOf("-Xjsr305=strict") // enable jsr305 null-safety in kotlin
-        jvmTarget = "1.8"
+        jvmTarget = "17"
     }
 }
 
 dependencies {
-    implementation("org.springframework.boot:spring-boot-starter-batch:2.7.1")
+    implementation("org.springframework.boot:spring-boot-starter-batch:3.0.0")
```

Comment:
Isn't it better to 3.0.0 as a symbol?

Reply:
I used hard coded one since it's a sample project.